### ISSUE TITLE
New feature and performance enhancement

### ISSUE THREAD

**Issue by gameboyVito** [ivpusic/react-native-image-crop-picker#388]
Reference Post #384 #385 
Dear author @ivpusic ,

We have discussed some new features listed below,
1. Added new properties localIdentifier and filename to be returned
2. Check if cropping and compression are both disabled. If yes, then do not return tmp path and generate copied files but only return localIdentifier. It is the essential to improve the performance when users want to select thousands of photos in the multiple selection mode.

The first one has been merged #385 
However, if someone don't know how to use the localIdentifier to get the `assetURL`. Here is an Example code:
```
/**
 * I write a litte util method to convert localIdentifier to assetURL in JavaScript
 * @param localIdentifier looks like 91B1C271-C617-49CE-A074-E391BA7F843F/L0/001
 * @param ext the extension: JPG, PNG, MOV
 * @returns {string}
 */
export const convertLocalIdentifierToAssetLibrary = (localIdentifier, ext) => {
    const hash =  localIdentifier.split('/')[0];
    return `assets-library://asset/asset.${ext}?id=${hash}&ext=${ext}`;
};
```

And the second one I would like to make a new PR.
Sent. #389

**Comment by ivpusic** [comment]
thanks. Isn't AlAssetLibrary and assets-library deprecated?

**Comment by gameboyVito** [comment]
No, the react-native `<Image />` component is still relying on this library. AlAssetLibrary is just not suggested to use after iOS8, but both AlAssetLibrary and PhotoKit are still available until now.
